extras/help_generator/writers.py

```diff
@@ -59,10 +59,10 @@ def write_help_html(doc_dic, helpdir, fname, sli_command_list, keywords):
     hlplist = []
     name = ''
 
-    for key, value in doc_dic.iteritems():
+    for key, value in doc_dic.items():
```

Comment:
I think wrapping with `iter()` will not provide any gains and rather make the code less readable. The only point of using `iteritems()` in Python 2 is to get a generator instead of a full list of items. `iter(d.items())` will still create the full list and then wrap an iterator around it in Python 2.  And `for k, v in d.items():` is the pythonic way as far as I know.